src/librustc_driver/driver.rs

```diff
@@ -378,9 +380,11 @@ impl<'a, 'b, 'ast, 'tcx> CompileState<'a, 'b, 'ast, 'tcx> {
                          krate: ast::Crate,
                          cstore: &'a CStore)
                          -> CompileState<'a, 'b, 'ast, 'tcx> {
+        let cloned_span = krate.span.clone();
```

Comment:
For the following reason related to "moved value", I'm thinking of keep the cloned_span, but remove span.clone().
i.e. let cloned_span = krate.span;